src/utest/mod.rs

```diff
@@ -45,7 +45,7 @@ pub fn runner(tests: &[&dyn Fn()]) {
     end_utests();
 }
 
-fn uassert_eq<T: PartialEq + core::fmt::Debug>(lhs: T, rhs: T, test_name: &str) {
+pub fn uassert_eq<T: PartialEq + core::fmt::Debug>(lhs: T, rhs: T, test_name: &str) {
```

Comment:
Is it necessary to expose the function? There are macros available IIRC

Reply:
For some reason macros doesn't work in I can't get access to this in src/allocator/mod.rs (see line 42). I will try to figure this out, but I just wanted to make something work x)

Reply:
I think you need to `use` the macros at the beginning of the file

```rust
use crate::kassert_eq;
use crate::kassert;
```

Reply:
I get this error:
```
error[E0425]: cannot find function `uassert_eq` in this scope
  --> src/utest/mod.rs:34:9
   |
34 |         uassert_eq($stmt, true, $name)
   |         ^^^^^^^^^^ not found in this scope
   |
  ::: src/spin_lock/mod.rs:58:13
   |
58 |             kassert!(*data, 1);
   |             ------------------- in this macro invocation
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider importing this function
   |
44 |     use crate::utest::uassert_eq;
   |
```

Reply:
Mmmh that's odd. I'm gonna take a look at it on your branch